sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -1004,7 +1016,8 @@ object CodeGenerator extends Logging {
    * weak keys/values and thus does not respond to memory pressure.
    */
   private val cache = CacheBuilder.newBuilder()
-    .maximumSize(100)
+    .maximumWeight(10 * 1024 * 1024)
```

Comment:
According to https://issues.apache.org/jira/browse/SPARK-18492, I assumed that Spark raises exceptions for the code with size 64KB.